drill-cicd-cli/build.gradle.kts

```diff
@@ -50,15 +49,16 @@ tasks {
 
 
 dependencies {
+    compileOnly((kotlin("stdlib-jdk8")))
+
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:$kotlinxCoroutinesVersion")
-    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core-jvm:$kotlinxCoroutinesVersion")
-    implementation("com.github.ajalt.clikt:clikt:3.5.4")
+    implementation("com.github.ajalt.clikt:clikt:3.3.0")
```

Comment:
What are the reasons for downgrading the library version?

Reply:
That was my mistake, version was changed back.